partiql-eval/src/test_value.rs

```diff
@@ -0,0 +1,194 @@
+use partiql_value::{EqualityValue, NullableEq, Value};
```

Comment:
seems like this file is copied directly from partiql-conformance-tests (other than the eval version including `parse_partiql_value_str`). do we want to consolidate the test value logic into one reusable struct at some point? if so, could add a todo

Reply:
Yes. I'd like to. Where, exactly, to put it is the question.

Reply:
maybe in `partiql-eval` is a better fit since it would be weird for `partiql-eval` to depend on `partiql-conformance-tests`? or possibly some other util/testing crate we create

Reply:
Yeah. I think a new util crate of some sort. I intend to work on it as part of the graph workstream.